Framework/API/test/LiveListenerTest.h

```diff
@@ -13,24 +13,29 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+namespace Poco::Net {
+// Poco does not not define this include the Poco::net
+inline std::ostream &operator<<(std::ostream &os, const Poco::Net::SocketAddress &) {
+  return (os << "Custom SocketAddress stream operator for GoogleMock");
```

Comment:
This will end up producing a confusing output when a mock fails, could we change it to something like:

```
inline std::ostream &operator<<(std::ostream &os, const Poco::Net::SocketAddress &obj) {
    return (os << "SocketAddress mock at: " << obj);
}
```

Reply:
Yes you're right. I was getting mildly irritated by having to implement these things by this point...

Reply:
I tried this originally to but some compilers can't figure out calling the real operator<< for SocketAddress. I'll add the same style as the one I did for `boost::optional`.